plasmapy/formulary/collisions.py

```diff
@@ -2033,3 +2064,271 @@ def coupling_parameter(
+        ...     "e-", "e-", v_a=v_a, n_b=n_b, T_b=T_b, Coulomb_log=Coulomb_log
+        ... )
+        >>> frequencies_using_velocity.energy_loss
+        <Quantity -9.69828719e+15 Hz>
+
+        See Also
+        --------
+        ~plasmapy.formulary.collisions.Coulomb_logarithm : Evaluates the Coulomb
+            logarithm for two interacting electron species.
+        """
+
+        # Note: This function uses CGS units internally to coincide with our references.
+        # Input is taken in MKS units and then converted as necessary. Output is in MKS units.
+
+        if v_a is None:
+            if T_a is not None:
+                v_a = thermal_speed(T_a, test_particle)
+            else:
+                raise ValueError("Please specify either v_a or T_a.")
+        elif T_a is not None:
+            raise ValueError("Please specify either v_a or T_a, not both.")
+
+        self.test_particle = test_particle
+        self.field_particle = field_particle
+        self.v_a = v_a
+        self.T_a = (
+            T_a
+            if T_a is not None
+            else (0.5 * test_particle.mass * v_a**2 / k_B).to(u.K)
+        )
+        self.n_b = n_b
+        self.T_b = T_b
+        self.Coulomb_log = Coulomb_log
+
+    @cached_property
+    def _mass_ratio(self):
+        return self.test_particle.mass / self.field_particle.mass
+
+    @cached_property
+    def momentum_loss(self):
+        """
+        The momentum loss rate due to collisions.
+        """
+        return (1 + self._mass_ratio) * self.phi * self.Lorentz_collision_frequency
+
+    @cached_property
+    def transverse_diffusion(self):
+        """
+        The rate of transverse diffusion due to collisions.
+        """
+        return (
+            2
+            * ((1 - 1 / (2 * self.x)) * self.phi + self._phi_prime)
+            * self.Lorentz_collision_frequency
+        )
+
+    @cached_property
+    def parallel_diffusion(self):
+        """
+        The rate of parallel diffusion due to collisions.
+        """
+        return (self.phi / self.x) * self.Lorentz_collision_frequency
+
+    @cached_property
+    def energy_loss(self):
+        """
+        The energy loss rate due to collisions.
+        """
+        return (
+            2
+            * (self._mass_ratio * self.phi - self._phi_prime)
+            * self.Lorentz_collision_frequency
+        )
+
+    @cached_property
+    def Lorentz_collision_frequency(self):
+        r"""
+        The Lorentz collision frequency (see Ch. 5 of :cite:t:`chen:2016`) is given
+        by
+
+        .. math::
+
+            ν = n σ v \ln{Λ}
+
+        where :math:`n` is the particle density, :math:`σ` is the
+        collisional cross-section, :math:`v` is the inter-particle velocity
+        (typically taken as the thermal velocity), and :math:`\ln{Λ}` is the
+        Coulomb logarithm accounting for small angle collisions.
+
+        See Equation (2.14) in :cite:t:`callen:unpublished`.
```

Comment:
I think this is rendering like this because the Callan book doesn't have a date attached to it?

![image](https://user-images.githubusercontent.com/32618747/186972314-551f1641-9759-4941-a739-dbaecbb32deb.png)

Reply:
This appears this way elsewhere in the docs, so I guess we can just leave it.